android/modules/ui/src/js/listview.js

```diff
@@ -83,26 +83,30 @@ exports.bootstrap = function (Titanium) {
 		}
 	}
 
+	function lookupProxyConstructor(namespace) {
+		const namespaceIndex = namespace.lastIndexOf('.');
+		const proxyName = namespace.slice(namespaceIndex + 1);
+		const parentNamespace = namespace.substring(0, namespaceIndex);
+		const segments = parentNamespace.split('.');
+		let parentProxy = global;
+		for (let i = 0; i < segments.length; i++) {
+			parentProxy = parentProxy[segments[i]];
+		}
+		const method = parentProxy['create' + proxyName];
+		if (parentProxy && method) {
```

Comment:
You're doing the `parentProxy` null check after using it.

Perhaps it should be coded like the below?
```javascript
let method = null;
if (parentProxy) {
	method = parentProxy['create' + proxyName];
}
if (!method) {
	throw new Error('Could not lookup constructor for namespace: "' + namespace + '"');
}
return method;
```

Reply:
Good catch!